src/serialization/tokenizer.cpp

```diff
@@ -135,16 +141,16 @@ const token &tokenizer::next_token()
 		[[fallthrough]];
 
 	default:
-		if (is_alnum(current_) || current_ == '$') {
+		if (is_alnum(current_) || current_ == token::DOLLAR) {
```

Comment:
Dollar… isn't even a token, really? (Also, I'm pretty sure it's not _supposed_ to be an identifier character… why was it added, again?)